database.js

```diff
@@ -5,12 +5,20 @@ const db = pgp(connectionString)
 
 const insertPost = 'INSERT INTO posts (title, written_at, body) VALUES ($1, now(), $2) RETURNING id'
 const selectPostById = 'SELECT * FROM posts WHERE id=$1'
+const selectAllPosts = 'SELECT * From Posts'
 
 const Post = {
-  insert: ({title, body}) => db.oneOrNone(insertPost, [title, body]),
-  select: (post_id) => db.oneOrNone(selectPostById, [post_id])
+  insert: ({title, body}) => db.oneOrNone('INSERT INTO posts \
+                                          (title, written_at, body) \
+                                          VALUES ($1, now(), $2) \
+                                          RETURNING id', [title, body]),
+  select: (post_id) => db.oneOrNone('SELECT * FROM posts WHERE id=$1', [post_id]),
+  selectAll: () => db.any('SELECT * From Posts', [])
```

Comment:
Stay consistent with your SQL syntax for readability. I see `FROM` and `From`, `Posts` and `posts` - I suggest that you pick the one that's closest to convention, and try to remain consistent with it throughout the rest of your project.